internal/lib/lib.go

```diff
@@ -651,7 +652,12 @@ func InformersToRegister(oclient *oshiftclient.Clientset, kclient *kubernetes.Cl
 		allInformers = append(allInformers, utils.NodeInformer)
 
 		informerTimeout := int64(120)
-		_, err := oclient.RouteV1().Routes("").List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &informerTimeout})
+		_, err := kclient.CoreV1().Services("").List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &informerTimeout})
+		if err != nil {
+			utils.AviLog.Warnf("can't access the services api for cluster %s, error : %s", err)
```

Comment:
Services API seems like services v2 apis - can we just say - Service object instead?

Reply:
Done